activerecord/test/cases/adapters/postgresql/range_test.rb

```diff
@@ -284,6 +284,26 @@ def test_timezone_awareness_tsrange
     end
   end
 
+  def test_timezone_awareness_unbounded_tsrange
+    tz = "Pacific Time (US & Canada)"
+
+    in_time_zone tz do
+      PostgresqlRange.reset_column_information
+      time_string = Time.current.to_s
+      time = Time.zone.parse(time_string)
```

Comment:
We can only use `Time.zone.now `& it will yield the same results instead of 
`time_string = Time.current.to_s
 time = Time.zone.parse(time_string)`
 
 Please check the screenshot attached below.   
<img width="405" alt="Screenshot 2022-10-28 at 20 46 44" src="https://user-images.githubusercontent.com/45059736/198710025-ab83f68a-ef7d-42ae-bfc4-bfe925a1362d.png">

Reply:
We also use `time_string` below, so it is needed.